src/components/EventCard/EventCard.tsx

```diff
@@ -0,0 +1,76 @@
+import { MouseEventHandler, useState } from "react";
+import "./EventCard.scss";
+import arrowUp from "../../assets/images/arrow-up.svg";
+import arrowDown from "../../assets/images/arrow-down.svg";
+import Gallery from "../Gallery/Gallery";
+import Button from "../Button/Button";
+
+type EventCardProps = {
+  title: string;
+  maker: string;
+  date: string;
+  textContent: string;
+  galleryArray: string[];
+  handleClick: MouseEventHandler<HTMLButtonElement>;
+};
+
+const EventCard = ({
+  title,
+  maker,
+  date,
+  textContent,
+  galleryArray,
+  handleClick,
+}: EventCardProps) => {
+  const [showInfo, setShowInfo] = useState<boolean>(false);
+
+  const toggleDisplay = () => {
+    if (showInfo) {
+      setShowInfo(false);
+    } else {
+      setShowInfo(true);
+    }
+  };
+
+  return (
+    <div className="event-card">
+      <div className="event-card__main">
+        <div className="event-card__header">
+          <h4 className="event-card__title">{title}</h4>
+          {!showInfo && (
+            <img
+              className="event-card__arrow"
+              src={arrowDown}
+              alt="arrow"
+              onClick={toggleDisplay}
+            />
+          )}
+          {showInfo && (
```

Comment:
This toggle functionality should be unit tested with RTL as the component is used in multiple places & is some core functionality that would be good to know if it gets broken at any point.